Datadog.Serverless/Datadog.Serverless.csproj

```
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <Version>0.1.0</Version>
+
+    <!-- NuGet package -->
+    <PackageId>Datadog.Serverless</PackageId>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <PackageIcon>packageIcon.png</PackageIcon>
+    <PackageProjectUrl>https://docs.datadoghq.com/serverless/</PackageProjectUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageTags>Datadog;APM;tracing;instrumentation;serverless</PackageTags>
+    <PackageDescription>Datadog Serverless Compat library for .NET.</PackageDescription>
+    <Copyright>Copyright 2025 Datadog, Inc.</Copyright>
+    <Company>Datadog</Company>
+    <Authors>Datadog</Authors>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+  </PropertyGroup>
+
+  <!-- Use deterministic build in CI -->
+  <PropertyGroup Condition="'$(GITLAB_CI)' == 'true' Or '$(GITHUB_ACTION)' == 'true'">
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)..\README.md" Pack="true" PackagePath="\" />
+    <None Include="$(MSBuildThisFileDirectory)..\datadog-logo-256x256.png" Pack="true"
+      Visible="false" PackagePath="packageIcon.png" />
+
+    <Content Include="datadog/bin/**">
```

Once we specify N numbers of targets in `<TargetFrameworks>`, by default these files will be copied N+1 times in the nuget package, packing the package larger. We can make sure the files are only copied twice by specifying paths explicitly.

```suggestion
    <Content Include="datadog/bin/**" Pack="true" PackagePath="content/datadog/bin;contentFiles/any/any/datadog/bin">
```

The `content/*` path is for older versions of nuget. `contentFiles/any/any/*` if for newer versions. If we don't do this, we end up with file copies in:
```
content/*
contentFiles/any/netstandard2.0/*
contentFiles/any/net461/*
```